src/app/home/home.component.html

```diff
@@ -0,0 +1,2 @@
+<p class="welcome">Welcome to the MSAL.js Angular Quickstart!</p>
```

Comment:
small suggestion: it would have been nice if there was a small indication on this page when the user successfully logs in, perhaps a message or a flashing icon.

Reply:
The login button changes to logout, but that's it. I'll consider doing more to make it more obvious.